cmd/controller/app/controller.go

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
is this a new file? I just surprised that git doesn't show it as a move with a minor tweak (if anything was changed).